bolt/test/AArch64/constant-island-entry.s

```diff
@@ -0,0 +1,35 @@
+// This test checks that we ignore functions which add an entry point that
+// is in a constant island.
+
+# RUN: llvm-mc -filetype=obj -triple aarch64-unknown-unknown %s -o %t.o
+# RUN: %clang %cflags %t.o -pie -Wl,-q -o %t.exe
+# RUN: llvm-bolt %t.exe -o %t.bolt 2>&1 | FileCheck %s
+
+# CHECK: BOLT-WARNING: Ignoring entry point at address 0x{{[0-9a-f]+}} in constant island of function func
+
+.globl func
+.type func, %function
+func:
+  ret
+  nop
```

Comment:
`ret` makes the next block unreachable. Is this needed to trigger the case or can we drop these two lines? Could also drop `main`.

Reply:
You're right, it's not needed. I've removed main as well. Thanks for pointing this out.